test/entrypoint.sh

```diff
@@ -36,3 +36,4 @@ parse_hacbs_test_output sanity_label_check conftest unittest.json
 
 echo "Starting Integeration-Tests"
 bats $POLICY_PATH/conftest.sh
+exec $@
```

Comment:
I don't understand purpose of this even after reading https://github.com/redhat-appstudio/hacbs-test/pull/20/files#r875791522

Reply:
I got it now and I don't like it

Reply:
https://github.com/redhat-appstudio/hacbs-test/pull/40/commits/904df2356b445415ff529a54f70481af5904961b I did something, please check if you like it